arviz/plots/backends/bokeh/ppcplot.py

```diff
@@ -260,18 +284,32 @@ def plot_ppc(
                     line_alpha=alpha,
                 )
                 glyph.level = "overlay"
+                legend_it.append((label, [glyph]))
 
+            all_scatter = []
             for vals, y in zip(pp_sampled_vals, y_rows[1:]):
                 vals = np.ravel(vals)
                 yvals = np.full_like(vals, y, dtype=np.float64)
                 if jitter:
                     yvals += np.random.uniform(low=scale_low, high=scale_high, size=len(vals))
-                ax_i.scatter(vals, yvals, fill_color=color, size=markersize, fill_alpha=alpha)
+                scatter = ax_i.scatter(
+                    vals, yvals, fill_color=color, size=markersize, fill_alpha=alpha
+                )
+                all_scatter.append(scatter)
 
+            legend_it.append(("{} predictive".format(group.capitalize()), all_scatter))
             ax_i.yaxis.major_tick_line_color = None
             ax_i.yaxis.minor_tick_line_color = None
             ax_i.yaxis.major_label_text_font_size = "0pt"
 
+        if legend:
+            legend = Legend(
+                items=legend_it,
+                location="center_right",
```

Comment:
given that the legend is fixed, I would place it on the top left, which will generally be en empty corner for both kde and cdf plots. https://arviz-devs.github.io/arviz/examples/plot_ppc_cumulative.html https://arviz-devs.github.io/arviz/examples/plot_ppc.html

Reply:
Done!
![Screenshot from 2021-03-06 15-18-12](https://user-images.githubusercontent.com/40103387/110202799-04c04800-7e91-11eb-9be7-47c38b7b18e2.png)